src/app/view/view.js

```diff
@@ -42,16 +42,16 @@ class View {
     this.render('.game__mode-rules', rules(mode));
   }
 
-  renderInitialScreen() {
-    // RENDER LOGO
-    // this.render('.header__game-logo', //createlogo);
-    // // RENDER GAME MODE BUTTONS
-    // this.render('.header__game-nav', //gamemodebuttons);
-    // // RENDER INIT LOOK OF GAME_MODE
-    // this.render('.game__mode', 'Choose category');
-    // // RENDER RULES
-    // this.render('.game__mode-rules', 'Harry Potter Quiz Game');
-  }
+  // renderInitialScreen() {
```

Comment:
If its not in the scope of your task, its best not to push this file.

Reply:
ups. pushing this file was out of my aware control

Reply:
done!